### PR TITLE
Update modules docs

### PR DESCRIPTION
**Pull request by stavares843**
Before

image

After

image

### CODE REVIEW

File: docs/content/getting-started/modules.md [hunk 1]
```diff
@@ -48,27 +48,27 @@ module github.com/user/repo
...
 
-You can use `hof mod` for generally managing CUE or Hof modules and dependencies.
+You can use `hof mod` to manage CUE or Hof modules and dependencies generally.
 In addition to setting, fetching, and validating dependencies,
-you can use `hof mod` to setup local development when working
+you can use `hof mod` to set up local development when working
 with multiple CUE or Hof modules.
 
-In your `cue.mods` file, use the `replace` directive just like you would in Go.
+Use the' replace' directive in your `cue.mods` file just like you would in Go.
```
Comment (stavares843) [inline docs/content/getting-started/modules.md]:
```suggestion
Use the `replace` directive in your `cue.mods` file just like you would in Go.
```

**Comment by verdverm** [comment]
This page is going to need a lot of work. I recently rewrote the command and I want to rethink how we talk about modules across 3 places:
- cuetorials
- this page
- a new top-level section

I'll start a new GDoc so we can collect our thoughts

**Comment by stavares843** [comment]
sounds good! 🔨

**Comment by stavares843** [comment]
closing for now, will reopen after we discuss it in the gdocs 🔨

**Pull request closed without merging**